starter-core/src/main/java/io/micronaut/starter/feature/function/gcp/template/gcpFunctionKotlinJunit.rocker.raw

```diff
@@ -18,8 +18,20 @@ class @project.getClassName()FunctionTest {
     @@Test
     fun testFunction() {
         HttpFunction().use { function ->
-           val response = function.invoke(HttpMethod.GET, "/@project.getPropertyName()")
+           val response = function.invoke(HttpMethod.GET, "/")
            Assertions.assertEquals(HttpStatus.OK, response.getStatus());
         }
     }
+
+
+    @@Test
+    fun testPost(){
+        HttpFunction().use { function ->
+            val input = SampleInputMessage()
+            input.name = "Test Name"
+            val request = HttpRequest.POST("/", input).contentType(MediaType.APPLICATION_JSON_TYPE)
```

Comment:
`.contentType(MediaType.APPLICATION_JSON_TYPE)` is unnecessary. Micronaut defaults to `application/json`